scss/settings/_component_variables.scss

```
@@ -150,6 +150,9 @@ $moo-switch-checked-bg-image-dark: url("data:image/svg+xml,<svg xmlns='http://ww
 // changes nothing but the chevron and revealed content.
 $accordion-button-active-bg: var(--bs-accordion-bg) !default;
 $accordion-button-active-color: var(--bs-accordion-color) !default;
+// A nested accordion inherits its Card surface by default, while remaining
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)